app/stac_api/validators.py

```diff
@@ -496,8 +496,8 @@ def validate_bbox(self, bbox):
         try:
             list_bbox_values = bbox.split(',')
             if (
-                list_bbox_values[0] == list_bbox_values[2] and
-                list_bbox_values[1] == list_bbox_values[3]
+                float(list_bbox_values[0]) == float(list_bbox_values[2]) and
+                float(list_bbox_values[1]) == float(list_bbox_values[3])
```

Comment:
Same as above, would keep the string comparaison here

Reply:
Oops forget about testing string equality, I didn't thought of this use case : `[6.10,47.10,6.1,47.1]` which would not pass the string equality but would pass the float equality !